tensorflow/lite/micro/micro_profiler.h

```diff
@@ -72,6 +76,14 @@ class MicroProfiler {
   uint32_t end_ticks_[kMaxEvents];
   int num_events_ = 0;
 
+  struct ticks_per_tag {
+    const char* tag;
+    uint32_t ticks;
+  };
+  ticks_per_tag total_ticks_per_tag[kMaxEvents];
+
+  int FindExistingOrNextPosition(const char* tagName);
```

Comment:
nit: add a comment about what this function does.

Reply:
Done. Thank you for the suggestion!